py/better_lotameapi.py

```diff
@@ -0,0 +1,209 @@
+'''
+    Please note that this file is an example, not an official Lotame-supported
+    tool. The Support team at Lotame does not provide support for this script,
+    as it's only meant to serve as a guide to help you use the Services API.
+
+    Filename: better_lotameapi.py
+    Author: Brett Coker
+    Version: 1.1.5
+    Python Version: 3.6.1
```

Comment:
Should this be 3.6.2?

Reply:
At this point it could be 3.6.5. I haven't made any updates to that file in awhile, and I guess the last time that I did, I was using 3.6.1.

Reply:
Ah. Was just seeing a discrepancy between that and the other files but I get it.